nitro-protocol/tsconfig.json

```diff
@@ -1,31 +1,13 @@
 {
-  "extends": "../../tsconfig.json",
   "compilerOptions": {
-    /* Basic Options */
-    "baseUrl": "src",
-    "lib": ["es5", "es6"],
-    "outDir": "lib" /* Redirect output structure to the directory. */,
-    "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
-    "target": "es5",
-    "strict": false /* Enable all strict type-checking options. */,
-    "removeComments": false,
-    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
-    "types": ["node", "jest"],
+    "target": "es2018",
+    "module": "commonjs",
+    "strict": true,
```

Comment:
Good to see `"strict": true`. Does `tsc .` work, or are there errors?

Reply:
`tsc --project ./` from root folder produces
```bash
Errors  Files
     7  node_modules/@types/jest/index.d.ts:35
     2  scripts/postdeploy.ts:6
```

Reply:
Maybe outside the scope of this work, but it might be nice to get to zero errors. Also, we want to gitignore the resulting `dist` folder.